### PR TITLE
fix(codewhisperer): Update keybinding for suggestion navigation

### PR DESCRIPTION
**Pull request by leigaol**
## Problem
 https://github.com/aws/aws-toolkit-vscode/issues/2939

## Solution
Only enable keybinding when CodeWhisperer is enabled.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by JadenSimon — APPROVED** [review]
Needs bugfix changelog

**Comment by leigaol** [comment]
> Needs bugfix changelog

Just added a new change log item.

**Review by JadenSimon — APPROVED** [review]

File: src/codewhisperer/activation.ts [hunk 1]
```diff
@@ -262,7 +262,9 @@ export async function activate(context: ExtContext): Promise<void> {
         updateCloud9TreeNodes.execute()
     } else if (isInlineCompletionEnabled()) {
         await setSubscriptionsforInlineCompletion()
-        await vscode.commands.executeCommand('setContext', 'CODEWHISPERER_ENABLED', true)
+        if (await getManualTriggerStatus()) {
+            await vscode.commands.executeCommand('setContext', 'CODEWHISPERER_ENABLED', true)
+        }
```
Comment (JadenSimon) [inline src/codewhisperer/activation.ts:267]: I'd remove the if statement as it introduces a bug.

Comment (JadenSimon) [reply src/codewhisperer/activation.ts:267]: Or you can move the context command somewhere else. Is there a single method for setting/checking CodeWhisperer enablement?

File: .changes/next-release/Bug Fix-291dc161-1ef0-4a28-9256-df2d9a5b5542.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Fixed bug when CodeWhisperer left/right arrow keybinding conflict with other plugins"
```
Comment (justinmk3) [inline .changes/next-release/Bug Fix-291dc161-1ef0-4a28-9256-df2d9a5b5542.json:3]: The type is "Bug Fix", so all that's needed is a description of the bug fix.

```suggestion
	"description": "CodeWhisperer left/right arrow keybinding conflict with other plugins"
```

**Comment by zixlin7** [comment]
tested the fix work as expected

**Review by zixlin7 — APPROVED** [review]

**Pull request merged**